app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,7 @@
   <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
   <%= csrf_meta_tags %>
   <%= favicon_link_tag 'favicon.ico' %>
+  <%= render 'layouts/google_analytics' %>
```

Comment:
Javascripts should usually go at the end of the document since they block the parser.

https://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup